hrabsence/hrabsence.php

```diff
@@ -235,6 +235,8 @@ function hrabsence_civicrm_install() {
   );
   civicrm_api3('message_template', 'create', $msg_params);
 
+  _hrabsence_add_scheduled_jobs();
```

Comment:
the upgrader install() method only work when installing the extension same as install hook .. so moving the code their won't screw anything .. but as I said i won't push on anything here since this is going to be replaced .. but let me know what you decided to do.